[BUG] `kbcli version` panic without k8s env

**Describe the bug**
`kbcli version` panic without k8s env or specify a wrong k8s config.

**To Reproduce**
Steps to reproduce the behavior:
1. `kbcli version`
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x28 pc=0x101de5ce0]

goroutine 1 [running]:
k8s.io/client-go/kubernetes.(*Clientset).AppsV1(0x103d95de0?)
	/home/runner/go/pkg/mod/k8s.io/client-go@v0.26.1/kubernetes/clientset.go:211
github.com/apecloud/kubeblocks/internal/cli/util.getKubeBlocksDeploy({0x103dbccb0?, 0x0?})
	/home/runner/work/kubeblocks/kubeblocks/internal/cli/util/version.go:101 +0x34
github.com/apecloud/kubeblocks/internal/cli/util.getKubeBlocksVersion({0x103dbccb0?, 0x0?})
	/home/runner/work/kubeblocks/kubeblocks/internal/cli/util/version.go:64 +0x24
github.com/apecloud/kubeblocks/internal/cli/util.GetVersionInfo({0x103dbccb0, 0x0})
	/home/runner/work/kubeblocks/kubeblocks/internal/cli/util/version.go:55 +0x138
github.com/apecloud/kubeblocks/internal/cli/cmd/version.(*versionOptions).Run(0x1400013c2e0, {0x103da9ac0?, 0x14000880700?})
	/home/runner/work/kubeblocks/kubeblocks/internal/cli/cmd/version/version.go:55 +0xd8
github.com/apecloud/kubeblocks/internal/cli/cmd/version.NewVersionCmd.func1(0x14000bf8300?, {0x102e1c295?, 0x0?, 0x0?})
	/home/runner/work/kubeblocks/kubeblocks/internal/cli/cmd/version/version.go:42 +0x28
github.com/spf13/cobra.(*Command).execute(0x14000bf8300, {0x1057b2fe0, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:920 +0x5b0
github.com/spf13/cobra.(*Command).ExecuteC(0x14000750600)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x35c
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
k8s.io/component-base/cli.run(0x14000750600)
	/home/runner/go/pkg/mod/k8s.io/component-base@v0.26.1/cli/run.go:146 +0x2a4
k8s.io/component-base/cli.RunNoErrOutput(...)
	/home/runner/go/pkg/mod/k8s.io/component-base@v0.26.1/cli/run.go:84
main.main()
	/home/runner/work/kubeblocks/kubeblocks/cmd/cli/main.go:28 +0x20
```

**Expected behavior**
kbcli version no panic without k8s env or specify a wrong k8s config.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS]
 - Browser [e.g. chrome, safari]
 - Version [e.g. 22]

**Additional context**
Add any other context about the problem here.

--- a/internal/cli/util/version.go
+++ b/internal/cli/util/version.go
@@ -19,6 +19,7 @@ package util
 import (
 	"context"
 	"fmt"
+	"reflect"
 
 	appsv1 "k8s.io/api/apps/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
@@ -44,7 +45,7 @@ func GetVersionInfo(client kubernetes.Interface) (map[AppName]string, error) {
 		KBCLIApp: version.GetVersion(),
 	}
 
-	if client == nil {
+	if client == nil || reflect.ValueOf(client).IsNil() {
 		return versionInfo, nil
 	}
 

--- a/internal/cli/util/version_test.go
+++ b/internal/cli/util/version_test.go
@@ -20,12 +20,33 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
+	"k8s.io/client-go/kubernetes"
+
 	"github.com/apecloud/kubeblocks/internal/cli/testing"
 )
 
 const kbVersion = "0.3.0"
 
 var _ = Describe("version util", func() {
+	It("get version info when client is nil", func() {
+		info, err := GetVersionInfo(nil)
+		Expect(err).Should(Succeed())
+		Expect(info).ShouldNot(BeEmpty())
+		Expect(info[KubeBlocksApp]).Should(BeEmpty())
+		Expect(info[KubernetesApp]).Should(BeEmpty())
+		Expect(info[KBCLIApp]).ShouldNot(BeEmpty())
+	})
+
+	It("get version info when client variable is a nil pointer", func() {
+		var client *kubernetes.Clientset
+		info, err := GetVersionInfo(client)
+		Expect(err).Should(Succeed())
+		Expect(info).ShouldNot(BeEmpty())
+		Expect(info[KubeBlocksApp]).Should(BeEmpty())
+		Expect(info[KubernetesApp]).Should(BeEmpty())
+		Expect(info[KBCLIApp]).ShouldNot(BeEmpty())
+	})
+
 	It("get version info when KubeBlocks is deployed", func() {
 		client := testing.FakeClientSet(testing.FakeKBDeploy(kbVersion))
 		info, err := GetVersionInfo(client)

